src/tests/source/wasm-particle-new.cc

```diff
@@ -1,4 +1,5 @@
 #include "src/wasm/cpp/arcs.h"
```

Comment:
What is the purpose of `wasm-particle-new.cc` vs `wasm-particle-old.cc`? This is the first time I've looked closely at these Wasm integration tests.

Reply:
They get built into two different wasm binaries, so the reload test can perform a genuine code swap.

Reply:
They get built into two different wasm binaries, so the reload test can perform a genuine code swap.